dnplab/__init__.py

```diff
@@ -7,4 +7,5 @@
 from . import hydrationGUI
 from . import core
 from . import dnpResults
+from .dnpIO import import_data
```

Comment:
How about `load_data`, `load` per easyspin style, or `read_data` or `read` per `nmrglue.bruker.read`, `pandas.read_XXX` ?

Reply:
I like calling the wrapper function "load".